docs/open.md

```diff
@@ -0,0 +1,55 @@
+# Artsy CLI Open
+
+Open Artsy links with iOS, Android or the browser.
+
+## Usage
+
+**With an alias:**
+
+```
+artsy open artist
+```
+
+opens https://artsy.net/artwork/banksy-love-rat-signed-16 on iOS
```

Comment:
I think this might be a typo? 

Looks like this actually opens "https://artsy.net/artist/banksy"